crates/swc_ecma_utils/src/lib.rs

```diff
@@ -1880,7 +1880,7 @@ impl Visit for LiteralVisitor {
             PropName::Str(ref s) => self.cost += 2 + s.value.len(),
             PropName::Ident(ref id) => self.cost += 2 + id.sym.len(),
             PropName::Num(n) => {
-                if n.value.fract() < 1e-10 {
+                if n.value.fract() < 1e-10 { // TODO: This breaks object simplifying. Why is this here?
```

Comment:
I think we may remove this

Reply:
`LiteralVisitor` is used to convert a javascript literal to `JSON.parse` when the object is huge, as `JSON.parse` is faster than javascript object literal.

I'm not sure why I implemented it in such a way. Maybe I just wanted to make it work for easy cases.